CMakeLists.txt

```diff
@@ -386,56 +379,40 @@ if(BUILD_TESTS)
       ${CMAKE_CURRENT_SOURCE_DIR}/src/node/rpc/test/frontend_test.cpp
     )
     target_link_libraries(
-      frontend_test PRIVATE ${CMAKE_THREAD_LIBS_INIT} lua.host http_parser.host
-                            sss.host ccf_endpoints.host quickjs.host
+      frontend_test PRIVATE ${CMAKE_THREAD_LIBS_INIT} http_parser.host sss.host
+                            ccf_endpoints.host quickjs.host
     )
 
     add_unit_test(
       tx_status_test
       ${CMAKE_CURRENT_SOURCE_DIR}/src/node/rpc/test/tx_status_test.cpp
     )
 
-    add_unit_test(
-      member_voting_test ${CMAKE_CURRENT_SOURCE_DIR}/src/js/wrap.cpp
-      ${CMAKE_CURRENT_SOURCE_DIR}/src/node/rpc/test/member_voting_test.cpp
-    )
-    target_link_libraries(
-      member_voting_test
-      PRIVATE ${CMAKE_THREAD_LIBS_INIT} lua.host http_parser.host sss.host
-              ccf_endpoints.host quickjs.host
-    )
-
-    set_tests_properties(
-      member_voting_test
-      PROPERTIES ENVIRONMENT
-                 RUNTIME_CONFIG_DIR=${CMAKE_SOURCE_DIR}/src/runtime_config
-    )
+    # TODO: Port or remove entirely? add_unit_test( member_voting_test
```

Comment:
I will have a look at porting what's missing from the end to end tests, unless there is a specific pattern we can't test there.